cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -823,13 +827,13 @@ private static CassandraNodeTask getMetadataTask(@NotNull final String executorI
         final Long availableMem = resourceValueLong(headOption(index.get("mem"))).or(0L);
         final Long availableDisk = resourceValueLong(headOption(index.get("disk"))).or(0L);
 
-        if (availableCpus <= cpu) {
-            errors.add(String.format("Not enough cpu resources for role %s. Required %f only %f available.", mesosRole, cpu, availableCpus));
+        if (availableCpus < cpu) {
```

Comment:
Nope - the unit test unveiled that exactly this scenario failed (offer==X and requirement==X). (We only want to fail, if the offer has less resources)